### DISCUSSION TITLE
Will dagsterApiGrpcArgs / codeServerArgs accept multiple --module-name arguments?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17526] [Q&A]
Heya good people!  Have a question about using the dagster-user-deployments Helm chart with multiple modules....
Will dagsterApiGrpcArgs / codeServerArgs accept multiple --module-name arguments?
We're installing multiple python packages containing assets, and attempting to load assets using --module-name args, but we're only seeing assets from  a single package in dagster-webserver.  All the packages appear to load assets independently, but no luck loading assets from multiple pkgs.  We can pivot to one container per module if need be, but thought I'd check here to see if I'm understanding things correctly.
I'd thought docs said the flag could be used multiple times, but now it appears that language is confined to arguments for dagster daemon in the documentation (I think).
Here's a excerpt from Helm values fwiw
```&gt; helm get values dagster-user-deployments -n dagster
USER-SUPPLIED VALUES:
deployments:
- affinity: {}
  annotations: {}
  dagsterApiGrpcArgs:
  - --module-name
  - dagster_demo
   - --module-name
  - de_dagsterbase_template
  - --module-name
  - de_dagster_rdcresidentialdata_fileland_etl
  - --inject-env-vars-from-instance```
Thanks a bunch for your time!

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C014N0PK37E/p1698769985625779?thread_ts=1698769985.625779&cid=C014N0PK37E).*

**Comment by gibsondan** [comment]
Unfortunately this is not currently possible. You can either have a separate deployment, one for each module, or a single entry point module that pulls in the definitions from each sub-module.

**Reply by de-j-magill** [reply]
Understood!  I think I conflated a couple of concepts, so that's on me.  It's easy enough to go either alternative route.  
I appreciate the information.  Thanks @gibsondan !
